ug-vmware-users/charter.md

```diff
@@ -0,0 +1,43 @@
+# VMware User Group Charter
+
+This User Group adheres to the governance described in [ug-governance] under the community standards of the Kubernetes project.
+
+## Scope
+The VMware User Group facilitates communication among Kubernetes users and contributors on topics pertaining to running all [conformant] forms of Kubernetes on VMware infrastructure. Scope includes extensions and integration of Kubernetes with Kubernetes subprojects.
+
+In addition to the usage of Kubernetes running on the cloud provider for the vSphere hypervisor, the User Group’s mission includes:
+
+- Running Kubernetes on desktop hypervisors (Fusion for OS X and Workstation for Linux and Windows), as supported by Minikube.
+- vCloud Director, which enables service providers to provision and manage multi tenant clouds, which can host Kubernetes.
+- Utilizing networking and storage features of VMware infrastructure through Kubernetes.
+- In the context of Kubernetes user use cases:
+  - Facilitating determination and documentation of best practices for configuring Kubernetes when running on vSphere infrastructure.
+  - Facilitating determination and documentation of best practices for configuring VMware infrastructure when supporting Kubernetes.
+  - Facilitating determination and documentation of best practices for using common configuration management, installation and orchestration tools for deploying and managing Kubernetes on VMware infrastructure.
+- Discussing bugs and feature requests recorded as Kubernetes, VMware cloud provider, or VMware storage plugin issues on GitHub. The UG itself is not responsible for actual management and resolution of issues, but merely facilitates members in creating well crafted issues in appropriate places - along with responses to comments and questions from prospective fixers within the issue.   
+
+## Out of Scope
+The VMware User Group does NOT:
+- Produce or own any Kubernetes code or project deliverables
+- Make Kubernetes project decisions
+- Claim ownership of any specific topic
+- Operate as a forum for discussing bugs outside the scope of Kubernetes and related open source cloud providers and plugins. For example, the VMware User Group should not be used to discuss or resolve support requests related to VMware products.
+ 
+## UG Example Activities
```

Comment:
I would recommend dropping this entirely as it's copied from https://github.com/kubernetes/community/blob/master/committee-steering/governance/ug-governance.md#ug-example-activities, I think it can be linked instead of copied